### PR TITLE
Added Like feature

### PR DESCRIPTION
**Pull request by sanjitsarkar**
In this PR,
I have added post liking feature

### CODE REVIEW

File: src/components/PostCard.jsx [hunk 1]
```diff
@@ -174,12 +184,48 @@ const PostCard = ({ post, type }) => {
...
       <div className="flex gap-x-10 gap-y-4 items-center flex-wrap">
         <div className="flex md:gap-3 gap-1 items-center md:flex-row flex-col">
-          <BiLike
-            size={25}
-            className="fill-primary cursor-pointer order-1 md:-order-1"
-          />
+          {!isPostLiked ? (
+            <MdFavoriteBorder
+              onClick={() => {
+                if (auth?.user?._id !== id) {
```
Comment (zoef-yt) [inline src/components/PostCard.jsx]: you can make auth?.user into a variable and use it as you are accessing it multiple times.

**Review by zoef-yt — APPROVED** [review]
The code looks good. small pr. You are good to merge.

File: src/components/PostCard.jsx (hunk 1 above)
Comment (sanjitsarkar) [reply src/components/PostCard.jsx]: Yes, It will be easier to access

**Review by JowelTisso — APPROVED** [review]
* Clean and modular code
* Good CSS classes
* Good use of asyncthunk
* Nice work with API call abstraction
* Looks good to merge

**Pull request merged**